client/css/text-target-length.css

```diff
@@ -18,3 +18,11 @@
 .target-length p.target-length-count.under b {
 	color: #e39d08;
 }
+.input-group p.target-length-count {
+    background: whitesmoke;
+    line-height: 27px;
```

Comment:
Should this use relative sizing or is this consistent with how the base field is styled in the CMS? If you increase the browser font size does it break?